parser/lexer.go

```diff
@@ -45,14 +48,11 @@ func (rr *runeReader) readRune() (r rune, size int, err error) {
 		return 0, 0, rr.err
 	}
 	r, sz := utf8.DecodeRune(rr.data[rr.pos:])
-	// TODO: Enable this check to make input strictly required to be UTF8. We may
-	//   want this to be an optional flag that the parser accepts, to make it
-	//   a conditional check. For now, since protoc allows bad UTF8, so must we :(
-	//if r == utf8.RuneError {
```

Comment:
There was a warning here due to missing whitespace before a comment. For now I just created a setting in the rune reader to enforce this check (but didn't plumb it all the way through).